src/app/public/plugin-resources/code-examples/description-list/horizontal/description-list-demo.component.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-description-list-demo',
+  templateUrl: './description-list-demo.component.html'
+})
+export class DescriptionListDemoComponent {
+
+  public items: { term: string, description: string }[] = [
+    {
+      term: 'College',
```

Comment:
Should either the horizontal or vertical examples include an example of a description that includes an unordered list? I didn't realize this was an option until reviewing the Design guidelines. Is it worth demonstrating in the code sample? Or is it simple enough that we don't need to demonstrate it?

Reply:
The unordered lists are separate. We'll have some styles for those soon. I don't think they wouldn't belong in these examples.